### PR TITLE
stout 1.2.2 (new formula)

### PR DESCRIPTION
**Pull request by renandincer**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

---

Stout is a deploy tool for static websites. It takes a website and uploads it to S3 in a more reliable way than other tools. It can be an alternative to paid services like Divshot, to dynamic web servers like Rails, or to manually uploading your site to S3 or an FTP server.

### CODE REVIEW

**Comment by renandincer** [comment]
EagerIO/Stout#16

File: Formula/stout.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+require "language/go"
+
+class Stout < Formula
+  desc "Reliable static website deploy tool"
+  homepage "http://stout.is"
+  url "https://github.com/EagerIO/Stout/archive/v1.2.2.tar.gz"
+  sha256 "e04a44735c4c04ccce63e73d211d2a8c0232594b23c8dfc5c971c10d1aa72aaa"
+  depends_on "go" => :build
```
Comment (DomT4) [inline Formula/stout.rb]: As much as I appreciate compactness, er, this is overkill 😄. As per the conventions of other formulae, please leave a single empty line between the end of each block, i.e.

```ruby

sha256 "e04a44735c4c04ccce63e73d211d2a8c0232594b23c8dfc5c971c10d1aa72aaa"

depends_on "go" => :build
depends_on "godep" => :build

def install
  blah
end

test do
  blah
end
```

File: Formula/stout.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
+require "language/go"
```
Comment (DomT4) [inline Formula/stout.rb]: You don't need this, since you're not using our `go_resource` system.

File: Formula/stout.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+class Stout < Formula
+  desc "Reliable static website deploy tool"
+  homepage "http://stout.is"
+  url "https://github.com/EagerIO/Stout/archive/v1.2.2.tar.gz"
+  sha256 "e04a44735c4c04ccce63e73d211d2a8c0232594b23c8dfc5c971c10d1aa72aaa"
+  depends_on "go" => :build
+  depends_on "godep" => :build
```
Comment (DomT4) [inline Formula/stout.rb]: If you're using the `vendor` system upstream we're more than happy to use the vendor system here. Our preference is more or less:

Vendor => Godeps => `go_resource`s, where all three are an option.

**Comment by renandincer** [comment]
👍

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-core/commit/2d78e4e7671a2410c0c5f0ccf5180578d3a0c312. Thank you for your contribution to Homebrew @renandincer; we appreciate it! :smiley_cat:

**Pull request closed without merging**